ports/stm32/Makefile

```diff
@@ -483,6 +483,14 @@ $(TOP)/lib/stm32lib/README.md:
 	$(ECHO) "stm32lib submodule not found, fetching it now..."
 	(cd $(TOP) && git submodule update --init lib/stm32lib)
 
+ifneq ($(FROZEN_MANIFEST),)
```

Comment:
Can this be combined, just so future changes don't accidentally miss updating one of the places?

Reply:
Done with trick